configure.ac

```diff
@@ -99,6 +99,7 @@ ZC_REQUIRE_TOOL(AR, ar)
 ZC_REQUIRE_TOOL(RANLIB, ranlib)
 ZC_REQUIRE_TOOL(STRIP, strip)
 ZC_REQUIRE_PROG([GIT], [git])
+ZC_REQUIRE_PROG(CLANG, clang)
```

Comment:
Why are we using the installed clang, rather than the pinned clang built under `depends`?